vnext/Microsoft.ReactNative/ReactApplication.h

```diff
@@ -81,15 +81,11 @@ struct ReactApplication : NoDefaultCtorReactApplication_base<ReactApplication> {
   void OnNavigationFailed(IInspectable const &, xaml::Navigation::NavigationFailedEventArgs const &);
 
  protected:
-  virtual ReactApplicationDelegate __stdcall CreateReactApplicationDelegate();
-
  private:
   ReactNative::ReactInstanceSettings m_instanceSettings{nullptr};
   Windows::Foundation::Collections::IVector<IReactPackageProvider> m_packageProviders{nullptr};
   ReactNative::ReactNativeHost m_host{nullptr};
 
-  ReactApplicationDelegate m_delegate{nullptr};
```

Comment:
> ReactApplicationDelegate m_delegate{nullptr}; [](start = 1, length = 46)

I think it is a good direction that we start to remove the ReactApplicationDelegate. It is better to have a simple ReactApplication and extend it. As a result the ReactApplicationDelegate code should be completely removed and good parts of it moved to ReactApplication.